src/components/Facility/FacilityForm.tsx

```diff
@@ -91,6 +93,20 @@ export default function FacilityForm({
     },
   });
 
+  const { data: org } = useQuery({
+    queryKey: ["organization", organizationId],
+    queryFn: query(organizationApi.get, {
+      pathParams: { id: organizationId },
+    }),
+    enabled: !!organizationId,
+  });
+
+  useEffect(() => {
+    const levels: Organization[] = [];
+    if (org && org.org_type === "govt") levels.push(org);
+    setSelectedLevels(levels);
+  }, [org, organizationId]);
```

Comment:
```suggestion
  }, [org]);
```

minor thing, but org Id isn't needed here.